### PR TITLE
Revamp docs for Docathon

### PR DESCRIPTION
**Pull request by metral**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!--Thanks for your contribution. See [CONTRIBUTING](CONTRIBUTING.md)
+    for Pulumi's contribution guidelines.
+
+    Help us merge your changes more quickly by adding more details such
```
Comment (lblackstone) [inline .github/PULL_REQUEST_TEMPLATE.md]: Is this something external contributors can set?

Comment (metral) [reply .github/PULL_REQUEST_TEMPLATE.md]: Hmm good point, I'm not sure. I copied this from https://github.com/pulumi/docs/blob/master/.github/ISSUE_TEMPLATE.md

Comment (bermudezmt) [reply .github/PULL_REQUEST_TEMPLATE.md]: They should be able to, but I haven't personally tested with a different account.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+```bash
+$ yarn link @pulumi/eks
+```
+
+## Running Integration Tests
+
+The examples and integration tests in this repository will create and destroy EKS clusters and Kubernetes objects while running.
+
+If possible, run your development PRs against the tests to confirm no errors
```
Comment (bermudezmt) [inline CONTRIBUTING.md]:
```suggestion
If possible, run your development PRs against the tests to confirm there are no errors
```

File: README.md [hunk 3]
```diff
@@ -4,67 +4,130 @@
 
 Pulumi's library for easily creating and managing EKS Kubernetes clusters.
 
-This package is meant for use with the Pulumi CLI.  Please visit [pulumi.io](https://pulumi.io) for
+This package is meant for use with the Pulumi CLI. Please visit [pulumi.io](https://pulumi.io) for
```
Comment (bermudezmt) [inline README.md]:
```suggestion
This package is meant for use with the Pulumi CLI. Please visit [pulumi.com](https://www.pulumi.com) for
```

File: README.md [hunk 4]
```diff
@@ -4,67 +4,130 @@
...
-- The cluster's worker nodes, which are managed by an autoscaling group
+- The EKS cluster
+- The cluster's worker nodes, and node groups which are managed by an autoscaling group
 - The Kubernetes resources needed to allow the worker nodes to access the cluster
-- Any Kubernetes StorageClasses needed for applications that will be deployed to the cluster
-- An optional deployment of the Kubernetes Dashboard
 
-The default configuration targets the default VPC and creates an autoscaling group of 2 "t2.medium" EC2 instances, a
-"gp2" default StorageClass, and a deployment of the Kubernetes Dashboard:
+The default configuration targets the AWS account's default VPC, and creates an autoscaling group of 2 `t2.medium` EC2 instances:
```
Comment (bermudezmt) [inline README.md]:
```suggestion
The default configuration targets the AWS account's default VPC, and creates an autoscaling group of two `t2.medium` EC2 instances:
```

File: README.md [hunk 5]
```diff
@@ -4,67 +4,130 @@
...
-const cluster = new eks.Cluster("cluster");
+// Create an EKS cluster with the default configuration.
+const cluster = new eks.Cluster("my-cluster");
 
+// Export the cluster's kubeconfig.
 export const kubeconfig = cluster.kubeconfig;
 ```
 
-Once the cluster is created, you can deploy into the cluster using "@pulumi/kubernetes", kubectl, helm, etc.
+Once the cluster is created, you can deploy into the cluster using the [`@pulumi/kubernetes`][pulumi-kubernetes] SDK, kubectl, Helm, etc.
```
Comment (bermudezmt) [inline README.md]:
```suggestion
Once the cluster is created, you can deploy to the cluster using the [`@pulumi/kubernetes`][pulumi-kubernetes] SDK, kubectl, Helm, etc.
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+The output will be stored in `/opt/pulumi/node_modules/@pulumi/eks`.
+
+```bash
+$ make only_build
+```
+
+There is also a helper [build script](./dev/build.sh) that restores the dependencies, builds, and installs the source.
+
+After `make` has completed, from your Pulumi program directory you must link the recent `@pulumi/eks` build from `/opt/` by running the following command:
```
Comment (bermudezmt) [inline CONTRIBUTING.md]:
```suggestion
`cd` into your Pulumi program directory.  After `make` has completed, link the recent `@pulumi/eks` build from `/opt/` by running the following command:
```

**Pull request merged**